django/src/rendering/views.py

```diff
@@ -0,0 +1,17 @@
+from django.template.loader import render_to_string
+from django.template import TemplateDoesNotExist
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework import status
+from .serializers import PageSerializer
+
+class FetchPageView(APIView):
+    def get(self, request, page_name):
+        try:
+            html_content = render_to_string(f'pages/{page_name}.html')
```

Comment:
Why putting templates in a pages folder?

Reply:
yes indeed not usefull